src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -749,3 +749,53 @@ def min_timestamp(aggregate_filter, org_id, use_case_id, alias=None):
 
 def max_timestamp(aggregate_filter, org_id, use_case_id, alias=None):
     return timestamp_column_snql("maxIf", aggregate_filter, org_id, use_case_id, alias)
+
+
+def http_error_count_span_snql(metric_column: str, tag_column: str, alias: str = None) -> Function:
+    return Function(
+        "countIf",
+        [
+            Column("value"),
+            Function(
+                "and",
+                [
+                    Function(
+                        "equals",
+                        [
+                            Column("metric_id"),
+                            metric_column,
```

Comment:
Is `metric_column` the equivalent of `aggregate_filter`? If so, you should use `aggregate_filter`, since it already injected by the layer.

Reply:
Where does that get injected? It's not an argument to my function.